articles/migrations/guides/calling-api-with-idtokens.md

```diff
@@ -0,0 +1,254 @@
+---
+title: Migration Guide: Management API and ID Tokens
+description: Auth0 is deprecating the usage of ID Tokens as credentials for the Management API. This article will help you migrate your solution from the old implementation to the new one.
+toc: true
+---
+
+# Migration Guide: Management API and ID Tokens
+
+For some use cases you could use [ID Tokens](/tokens/id-token) as credentials in order to call some of the endpoints of the [Management API](/api/management/v2). This functionality is being deprecated. You will have to use proper [Access Tokens](/tokens/access-token) in order to access any of the endpoints of the [Management API](/api/management/v2).
+
+This article will help you migrate your solution from the old implementation to the new one. First, we will see what this migration is about and which use cases are affected. We will continue with reviewing how you can use [scopes](/scopes) to get tokens with different access rights, and then see all the ways you can use to get an Access Token. Finally, we will review the [Account Linking](/link-accounts) process and the changes introduced to it.
+
+## Overview
+
+Some of the [/users](/api/management/v2#!/Users/get_users_by_id) and [/device-credentials](/api/management/v2#!/Device_Credentials/get_device_credentials) endpoints, could be called using ID Tokens. You can now use Access Tokens to access them. The grace period for this migration has already started and will end on the `1st of June, 2018`, when you will no longer be able to use ID Tokens as credentials for the Management API.
```

Comment:
We need to generate the final timeline.

Reply:
Do you want me to update to something generic (like Q3 2018) or will we have a specific date these coming days?